tangorb/common/src/main/java/fr/esrf/TangoApi/DbDevExportInfo.java

```diff
@@ -85,10 +62,7 @@ public DbDevExportInfo() {
     //===============================================
     public DbDevExportInfo(String name, String ior,
                            String host, String version) {
-        this.name = name;
-        this.ior = ior;
-        this.host = host;
-        this.version = version;
+        this.value = new DeviceExportInfo(name, ior, host, version, "", "");
```

Comment:
When I was mentioning the "null" string for the PID parameter, I meant a string with the word "null" in it.
This is what is expecting DbExportDevice command from the Database server when the the PID should not be taken into account.
See [Database.cpp](https://github.com/tango-controls/TangoDatabase/blob/master/DataBase.cpp#L1840)